src/icon-button.css.ts

```diff
@@ -1,7 +1,513 @@
+            [background]: skinVars.colors.buttonPrimaryBackground,
+            [backgroundHover]: skinVars.colors.backgroundContainerBrandHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerBrandPressed,
+            [iconColor]: skinVars.colors.textButtonPrimary,
+        },
+    },
+
+    'brand-soft-default': {
+        vars: {
+            [background]: skinVars.colors.brandLow,
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.textButtonSecondary,
+        },
+    },
+    'brand-soft-inverse': {
+        vars: {
+            [background]: skinVars.colors.backgroundContainerBrandOverInverse,
+            [backgroundHover]: skinVars.colors.backgroundContainerBrandHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerBrandPressed,
+            [iconColor]: skinVars.colors.textButtonSecondaryInverse,
+        },
+    },
+    'brand-soft-alternative': {
+        vars: {
+            [background]: skinVars.colors.brandLow,
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.textButtonSecondary,
+        },
+    },
+
+    'brand-default-default': {
+        vars: {
+            [background]: 'transparent',
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.textButtonSecondary,
+        },
+    },
+    'brand-default-inverse': {
+        vars: {
+            [background]: 'transparent',
+            [backgroundHover]: skinVars.colors.backgroundContainerBrandHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerBrandPressed,
+            [iconColor]: skinVars.colors.textButtonSecondaryInverse,
+        },
+    },
+    'brand-default-alternative': {
+        vars: {
+            [background]: 'transparent',
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: skinVars.colors.textButtonSecondary,
+        },
+    },
+
+    'neutral-solid-default': {
+        vars: {
+            [background]: skinVars.colors.neutralHigh,
+            [backgroundHover]: skinVars.colors.backgroundContainerBrandHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerBrandPressed,
+            [iconColor]: skinVars.colors.neutralLow,
+        },
+    },
+    'neutral-solid-inverse': {
+        vars: {
+            [background]: skinVars.colors.inverse,
+            [backgroundHover]: skinVars.colors.backgroundContainerHover,
+            [backgroundActive]: skinVars.colors.backgroundContainerPressed,
+            [iconColor]: 'black',
```

Comment:
why black? 🤔

Reply:
This is the same approach we're using in the current top actions definition, since inverse backgrounds are always white no matter the color-scheme, there's no available token that stays dark in both scenarios and is semantically suitable for this use case.